app/controllers/api/v1/clubs_controller.rb

```diff
@@ -12,6 +12,10 @@ def create
     render json: ClubSerializer.new(club), status: 201
   end
 
+  def destroy
+    render json: Club.delete(params[:id]), status: 204
```

Comment:
club = Club.find(params[:id]) 
render json: Club.delete(club), status: 204 

This will rescue you with a 404 error if the club you're trying to delete does not exist